parts/configure-swarmmode-cluster.sh

```diff
@@ -150,9 +151,13 @@ echo "Installing and configuring Docker"
 installDocker()
 {
   for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh
+    curl --max-time 60 -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - 

```

Comment:
Should it also install packages to allow apt to use repository over HTTPS? Or is does it come with the package already installed?
E.g.:
https://docs.docker.com/cs-engine/1.13/#install-on-ubuntu-1404-lts-or-1604-lts